qiskit_aqua/utils/run_circuits.py

```diff
@@ -293,3 +322,29 @@ def compile_and_run_circuits(circuits, backend, backend_config, compile_config,
     result = _combine_result_objects(results) if len(results) != 0 else None
 
     return result
+
+# Skip_validation = True does what backend.run and aerjob.submit do, but without qobj validation.
+def run_on_backend(backend, qobj, backend_options = None, noise_config = None, skip_validation = False):
```

Comment:
for setting default argument, you do not need space between `=` sign

Reply:
Sure, done